<launch>
  <param name="/use_sim_time" value="true" />
  <include file="$(find vp6242_arm_navigation)/launch/vp6242_planning_environment.launch" />
  <node pkg="planning_environment" name="wall_clock_server" type="fake_time.py" />
  
  <include file="$(find planning_environment)/launch/environment_server.launch">
  	<arg name="use_monitor" value="false" />
    <arg name="use_collision_map" value="false" />
  </include>
  
  <include file="$(find vp6242_arm_navigation)/launch/ompl_planning.launch" />
  
  <include file="$(find vp6242_arm_navigation)/launch/constraint_aware_kinematics.launch" />
    
  <node pkg="hg_cartesian_trajectory" type="cartesian_trajector_planner" name="cartesian_planner" output="screen">
  	<rosparam command="load" file="$(find vp6242_arm_navigation)/config/cartesian_planning.yaml" />
  </node>
</launch>